docs/fonts/glyphicons-halflings-regular.svg

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
is the usage for all fonts/image resources copy-right permitted? where are these from?

Reply:
DocStrap is based off of Bootstrap. Within Bootstrap's components are the set of Glyphicons that you are seeing in the `fonts` folder. Boostrap states [here](https://getbootstrap.com/docs/3.3/components/#glyphicons-glyphs) that these glyphs are free to use.